biz.aQute.bndlib/src/aQute/bnd/osgi/Contracts.java

```diff
@@ -56,7 +56,7 @@ public Contracts(Analyzer analyzer) {
 
 	Instructions getFilter() {
 		if (instructions == null) {
-			String contract = analyzer.getProperty(Constants.CONTRACT);
+			String contract = analyzer.getProperty(Constants.CONTRACT, "*");
```

Comment:
This PR should also update https://bnd.bndtools.org/instructions/contract.html to indicate that `*` is now the default.